src/lib/fetch.ts

```diff
@@ -38,25 +38,35 @@ export {
   setMinaGraphqlFallbackEndpoints,
   setArchiveGraphqlEndpoint,
   setArchiveGraphqlFallbackEndpoints,
+  setAccountsManagerEndpoint,
   sendZkappQuery,
   sendZkapp,
   removeJsonQuotes,
   fetchEvents,
   fetchActions,
+  KeyPair,
+  acquireKeyPair,
+  releaseKeyPair
 };
 
 type NetworkConfig = {
   minaEndpoint: string;
   minaFallbackEndpoints: string[];
   archiveEndpoint: string;
   archiveFallbackEndpoints: string[];
+  accountsManagerEndpoint: string;
```

Comment:
I'd advocate for making this name even more explicit to avoid confusing people. Like `lightnetAccountManagerEndpoint`